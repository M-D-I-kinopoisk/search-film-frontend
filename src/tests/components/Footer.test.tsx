import {fireEvent, render, screen} from '@testing-library/react'
import Footer from '../../components/Footer/Footer'
import {NextIntlProvider} from 'next-intl'

describe('Footer', () => {
    it('"Написать в чате" button renders ', () => {
        const component = render(
            <NextIntlProvider messages={{
                'footer': {
                    'item1': {
                        'title': 'Мой Иви',
                        'text1': 'О компании',
                        'text2': 'Вакансии',
                        'text3': 'Программа бета-тестирования',
                        'text4': 'Информация для партнёров',
                        'text5': 'Размещение рекламы',
                        'text6': 'Пользовательское соглашение',
                        'text7': 'Политика конфиденциальности',
                        'text8': 'Комплаенс'
                    },
                    'item2': {
                        'title': 'Разделы',
                        'text1': 'Мой Иви',
                        'text2': 'Что нового',
                        'text3': 'Фильмы',
                        'text4': 'Сериалы',
                        'text5': 'Мультфильмы',
                        'text6': 'TV+',
                        'text7': 'Что посмотреть',
                        'text8': 'Активация сертификата'
                    },
                    'item3': {
                        'title': 'Служба поддержки',
                        'text1': 'Мы всегда готовы вам помочь.',
                        'text2': 'Наши операторы онлайн 24/7',
                        'text3': 'Написать в чате',
                        'text4': 'Ответы на вопросы'
                    },
                    'item4': {
                        'title': 'Смотрите фильмы, сериалы и мультфильмы без рекламы'
                    },
                    'item5': {
                        'text1': 'Загрузить в',
                        'text2': 'Доступно в',
                        'text3': 'Смотрите на',
                        'text4': 'Все устройства'
                    }
                },
            }} locale={'ru'}>
                <Footer/>
            </NextIntlProvider>)
        expect(component).toMatchSnapshot()
        const chatBtn = screen.getByText('Написать в чате')
        expect(chatBtn).toBeInTheDocument()
    })
})