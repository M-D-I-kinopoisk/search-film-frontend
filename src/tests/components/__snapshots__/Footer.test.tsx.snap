// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer "Написать в чате" button renders  1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="wrapper"
      >
        <div
          class="footer"
        >
          <div
            class="footer__topBlock"
          >
            <div
              class="footer__column"
            >
              <p
                class="footer__title"
              >
                Мой Иви
              </p>
              <a
                class="footer__link"
                href="https://corp.ivi.ru/?_gl=1%2Alvn2xf%2A_ga%2AMTMxNTY0MzcwMS4xNjgxMjE0Mzc5%2A_ga_GETQ4387MJ%2AMTY4MTQ4MjkyMy42LjEuMTY4MTUwNzAzNy42MC4wLjA."
              >
                О компании
              </a>
              <a
                class="footer__link"
                href="https://corp.ivi.ru/career/#career-vacancy-block"
              >
                Вакансии
              </a>
              <a
                class="footer__link"
                href="https://www.ivi.ru/pages/beta/"
              >
                Программа бета-тестирования
              </a>
              <a
                class="footer__link"
                href="https://www.ivi.ru/info/partners"
              >
                Информация для партнёров
              </a>
              <a
                class="footer__link"
                href="https://corp.ivi.ru/advertisers/"
              >
                Размещение рекламы
              </a>
              <a
                class="footer__link"
                href="https://www.ivi.ru/info/agreement"
              >
                Пользовательское соглашение
              </a>
              <a
                class="footer__link"
                href="https://www.ivi.ru/info/confidential"
              >
                Политика конфиденциальности
              </a>
              <a
                class="footer__link"
                href="https://www.ivi.ru/info/goryachaya-liniya-komplaens"
              >
                Комплаенс
              </a>
            </div>
            <div
              class="footer__column"
            >
              <p
                class="footer__title"
              >
                Разделы
              </p>
              <a
                class="footer__link"
                href="/"
              >
                Мой Иви
              </a>
              <a
                class="footer__link"
                href="https://www.ivi.ru/new"
              >
                Что нового
              </a>
              <a
                class="footer__link"
                href="https://www.ivi.ru/movies"
              >
                Фильмы
              </a>
              <a
                class="footer__link"
                href="https://www.ivi.ru/series"
              >
                Сериалы
              </a>
              <a
                class="footer__link"
                href="https://www.ivi.ru/animation"
              >
                Мультфильмы
              </a>
              <a
                class="footer__link"
                href="https://www.ivi.ru/tvplus"
              >
                TV+
              </a>
              <a
                class="footer__link"
                href="https://www.ivi.ru/goodmovies"
              >
                Что посмотреть
              </a>
              <a
                class="footer__link gradient "
                href="https://www.ivi.ru/cert"
              >
                Активация сертификата
              </a>
            </div>
            <div
              class="footer__column"
            >
              <p
                class="footer__title"
              >
                Служба поддержки
              </p>
              <span>
                Мы всегда готовы вам помочь.
              </span>
              <span>
                Наши операторы онлайн 24/7
              </span>
              <button
                class="footer__btnChat"
              >
                Написать в чате
              </button>
              <div
                class="footer__btnGroup"
              >
                <button
                  class="footer__btn"
                >
                  <svg
                    fill="currentColor"
                    height="16"
                    stroke="currentColor"
                    stroke-width="0"
                    viewBox="0 0 14 16"
                    width="16"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M0 4v8c0 .55.45 1 1 1h12c.55 0 1-.45 1-1V4c0-.55-.45-1-1-1H1c-.55 0-1 .45-1 1zm13 0L7 9 1 4h12zM1 5.5l4 3-4 3v-6zM2 12l3.5-3L7 10.5 8.5 9l3.5 3H2zm11-.5l-4-3 4-3v6z"
                      fill-rule="evenodd"
                    />
                  </svg>
                </button>
                <button
                  class="footer__btn"
                >
                  <svg
                    fill="currentColor"
                    height="16"
                    stroke="currentColor"
                    stroke-width="0"
                    viewBox="0 0 512 512"
                    width="16"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M451 374c-15.88-16-54.34-39.35-73-48.76-24.3-12.24-26.3-13.24-45.4.95-12.74 9.47-21.21 17.93-36.12 14.75s-47.31-21.11-75.68-49.39-47.34-61.62-50.53-76.48 5.41-23.23 14.79-36c13.22-18 12.22-21 .92-45.3-8.81-18.9-32.84-57-48.9-72.8C119.9 44 119.9 47 108.83 51.6A160.15 160.15 0 0083 65.37C67 76 58.12 84.83 51.91 98.1s-9 44.38 23.07 102.64 54.57 88.05 101.14 134.49S258.5 406.64 310.85 436c64.76 36.27 89.6 29.2 102.91 23s22.18-15 32.83-31a159.09 159.09 0 0013.8-25.8C465 391.17 468 391.17 451 374z"
                      fill="none"
                      stroke-miterlimit="10"
                      stroke-width="32"
                    />
                  </svg>
                </button>
              </div>
              <div
                class="footer__groupQuestions"
              >
                <a
                  class="footer__linkAsk"
                  href="https://ask.ivi.ru/?_gl=1*fa2mrd*_ga*MTMxNTY0MzcwMS4xNjgxMjE0Mzc5*_ga_GETQ4387MJ*MTY4MTU2Njc5MS44LjEuMTY4MTU2NzI1My4zLjAuMA.."
                >
                  ask.ivi.ru
                </a>
                <span>
                  Ответы на вопросы
                </span>
              </div>
            </div>
            <div
              class="footer__column"
            >
              <a
                class="footer__columnRight"
                href="https://www.ivi.ru/subscribe?redirect_url=%2Fwatch%2F504525"
              >
                <div
                  class="footer__widget"
                >
                  <svg
                    color="#fff"
                    fill="currentColor"
                    height="60"
                    stroke="currentColor"
                    stroke-width="0"
                    style="color: rgb(255, 255, 255);"
                    viewBox="0 0 24 24"
                    width="60"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M0 0h24v24H0z"
                      fill="none"
                    />
                    <path
                      d="M16.5 12A4.5 4.5 0 0014 7.97v2.21l2.45 2.45c.03-.2.05-.41.05-.63zm2.5 0c0 .94-.2 1.82-.54 2.64l1.51 1.51A8.796 8.796 0 0021 12c0-4.28-2.99-7.86-7-8.77v2.06c2.89.86 5 3.54 5 6.71zM4.27 3L3 4.27 7.73 9H3v6h4l5 5v-6.73l4.25 4.25c-.67.52-1.42.93-2.25 1.18v2.06a8.99 8.99 0 003.69-1.81L19.73 21 21 19.73l-9-9L4.27 3zM12 4L9.91 6.09 12 8.18V4z"
                    />
                  </svg>
                </div>
                <span
                  class="footer__widgetText"
                >
                  Смотрите фильмы, сериалы и мультфильмы без рекламы
                </span>
              </a>
            </div>
          </div>
          <div
            class="footer__downBlock"
          >
            <div
              class="footer__downBlock_left"
            >
              <div
                class="footer__downBlock_group"
              >
                <a
                  class="footer__smartBtn"
                  href="https://apps.apple.com/RU/app/id455705533?mt=8"
                >
                  <img
                    alt="apple"
                    data-nimg="1"
                    decoding="async"
                    height="20"
                    loading="lazy"
                    src="https://solea-parent.dfs.ivi.ru/picture/ffffff,ffffff/appleLogo.svg"
                    style="color: transparent;"
                    width="20"
                  />
                  <div>
                    <p
                      class="footer__smartBtn_title"
                    >
                      Загрузить в
                    </p>
                    <p
                      class="footer__smartBtn_text"
                    >
                      App Store
                    </p>
                  </div>
                </a>
                <a
                  class="footer__smartBtn"
                  href="https://apps.apple.com/RU/app/id455705533?mt=8"
                >
                  <img
                    alt="apple"
                    data-nimg="1"
                    decoding="async"
                    height="20"
                    loading="lazy"
                    src="https://solea-parent.dfs.ivi.ru/picture/ffffff,ffffff/googlePlayLogo.svg"
                    style="color: transparent;"
                    width="20"
                  />
                  <div>
                    <p
                      class="footer__smartBtn_title"
                    >
                      Доступно в
                    </p>
                    <p
                      class="footer__smartBtn_text"
                    >
                      Goggle Play
                    </p>
                  </div>
                </a>
                <a
                  class="footer__smartBtn"
                  href="https://www.ivi.ru/pages/tvsmart/"
                >
                  <svg
                    fill="currentColor"
                    height="20"
                    stroke="currentColor"
                    stroke-width="0"
                    viewBox="0 0 24 24"
                    width="20"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M0 0h24v24H0z"
                      fill="none"
                    />
                    <path
                      d="M20 3H4c-1.1 0-2 .9-2 2v11c0 1.1.9 2 2 2h6l-2 2v1h8v-1l-2-2h6c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2"
                    />
                  </svg>
                  <div>
                    <p
                      class="footer__smartBtn_title"
                    >
                      Смотрите на
                    </p>
                    <p
                      class="footer__smartBtn_text"
                    >
                      Smart TV
                    </p>
                  </div>
                </a>
                <a
                  class="footer__smartBtn"
                  href="https://apps.apple.com/RU/app/id455705533?mt=8"
                >
                  <img
                    alt="apple"
                    data-nimg="1"
                    decoding="async"
                    height="20"
                    loading="lazy"
                    src="https://solea-parent.dfs.ivi.ru/picture/ffffff,ffffff/appleLogo.svg"
                    style="color: transparent;"
                    width="20"
                  />
                  <div>
                    <p
                      class="footer__smartBtn_text"
                    >
                      Все устройства
                    </p>
                  </div>
                </a>
              </div>
              <div>
                <p>
                  © 2023 ООО «Иви.ру»
                </p>
                <p>
                  HBO ® and related service marks are the property of Home Box Office, Inc
                </p>
              </div>
            </div>
            <div
              class="footer_downBlock_right"
            >
              <button
                class="footer__btn border"
              >
                <img
                  alt="vk"
                  data-nimg="1"
                  decoding="async"
                  height="16"
                  loading="lazy"
                  src="https://solea-parent.dfs.ivi.ru/picture/ffffff,ffffff/social_vkontakte.svg"
                  style="color: transparent;"
                  width="16"
                />
              </button>
              <button
                class="footer__btn border"
              >
                <img
                  alt="od"
                  data-nimg="1"
                  decoding="async"
                  height="16"
                  loading="lazy"
                  src="https://solea-parent.dfs.ivi.ru/picture/ffffff,ffffff/social_odnoklassniki.svg"
                  style="color: transparent;"
                  width="16"
                />
              </button>
              <button
                class="footer__btn border"
              >
                <img
                  alt="twitter"
                  data-nimg="1"
                  decoding="async"
                  height="16"
                  loading="lazy"
                  src="https://solea-parent.dfs.ivi.ru/picture/ffffff,ffffff/social_twitter.svg"
                  style="color: transparent;"
                  width="16"
                />
              </button>
              <button
                class="footer__btn border"
              >
                <img
                  alt="viber"
                  data-nimg="1"
                  decoding="async"
                  height="16"
                  loading="lazy"
                  src="https://solea-parent.dfs.ivi.ru/picture/ffffff,ffffff/social_viber.svg"
                  style="color: transparent;"
                  width="16"
                />
              </button>
              <button
                class="footer__btn border"
              >
                <img
                  alt="linkendin"
                  data-nimg="1"
                  decoding="async"
                  height="16"
                  loading="lazy"
                  src="https://solea-parent.dfs.ivi.ru/picture/ffffff,ffffff/social_linkedin.svg"
                  style="color: transparent;"
                  width="16"
                />
              </button>
              <button
                class="footer__btn border"
              >
                <img
                  alt="telegram"
                  data-nimg="1"
                  decoding="async"
                  height="16"
                  loading="lazy"
                  src="https://solea-parent.dfs.ivi.ru/picture/ffffff,ffffff/social_telegram.svg"
                  style="color: transparent;"
                  width="16"
                />
              </button>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="wrapper"
    >
      <div
        class="footer"
      >
        <div
          class="footer__topBlock"
        >
          <div
            class="footer__column"
          >
            <p
              class="footer__title"
            >
              Мой Иви
            </p>
            <a
              class="footer__link"
              href="https://corp.ivi.ru/?_gl=1%2Alvn2xf%2A_ga%2AMTMxNTY0MzcwMS4xNjgxMjE0Mzc5%2A_ga_GETQ4387MJ%2AMTY4MTQ4MjkyMy42LjEuMTY4MTUwNzAzNy42MC4wLjA."
            >
              О компании
            </a>
            <a
              class="footer__link"
              href="https://corp.ivi.ru/career/#career-vacancy-block"
            >
              Вакансии
            </a>
            <a
              class="footer__link"
              href="https://www.ivi.ru/pages/beta/"
            >
              Программа бета-тестирования
            </a>
            <a
              class="footer__link"
              href="https://www.ivi.ru/info/partners"
            >
              Информация для партнёров
            </a>
            <a
              class="footer__link"
              href="https://corp.ivi.ru/advertisers/"
            >
              Размещение рекламы
            </a>
            <a
              class="footer__link"
              href="https://www.ivi.ru/info/agreement"
            >
              Пользовательское соглашение
            </a>
            <a
              class="footer__link"
              href="https://www.ivi.ru/info/confidential"
            >
              Политика конфиденциальности
            </a>
            <a
              class="footer__link"
              href="https://www.ivi.ru/info/goryachaya-liniya-komplaens"
            >
              Комплаенс
            </a>
          </div>
          <div
            class="footer__column"
          >
            <p
              class="footer__title"
            >
              Разделы
            </p>
            <a
              class="footer__link"
              href="/"
            >
              Мой Иви
            </a>
            <a
              class="footer__link"
              href="https://www.ivi.ru/new"
            >
              Что нового
            </a>
            <a
              class="footer__link"
              href="https://www.ivi.ru/movies"
            >
              Фильмы
            </a>
            <a
              class="footer__link"
              href="https://www.ivi.ru/series"
            >
              Сериалы
            </a>
            <a
              class="footer__link"
              href="https://www.ivi.ru/animation"
            >
              Мультфильмы
            </a>
            <a
              class="footer__link"
              href="https://www.ivi.ru/tvplus"
            >
              TV+
            </a>
            <a
              class="footer__link"
              href="https://www.ivi.ru/goodmovies"
            >
              Что посмотреть
            </a>
            <a
              class="footer__link gradient "
              href="https://www.ivi.ru/cert"
            >
              Активация сертификата
            </a>
          </div>
          <div
            class="footer__column"
          >
            <p
              class="footer__title"
            >
              Служба поддержки
            </p>
            <span>
              Мы всегда готовы вам помочь.
            </span>
            <span>
              Наши операторы онлайн 24/7
            </span>
            <button
              class="footer__btnChat"
            >
              Написать в чате
            </button>
            <div
              class="footer__btnGroup"
            >
              <button
                class="footer__btn"
              >
                <svg
                  fill="currentColor"
                  height="16"
                  stroke="currentColor"
                  stroke-width="0"
                  viewBox="0 0 14 16"
                  width="16"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M0 4v8c0 .55.45 1 1 1h12c.55 0 1-.45 1-1V4c0-.55-.45-1-1-1H1c-.55 0-1 .45-1 1zm13 0L7 9 1 4h12zM1 5.5l4 3-4 3v-6zM2 12l3.5-3L7 10.5 8.5 9l3.5 3H2zm11-.5l-4-3 4-3v6z"
                    fill-rule="evenodd"
                  />
                </svg>
              </button>
              <button
                class="footer__btn"
              >
                <svg
                  fill="currentColor"
                  height="16"
                  stroke="currentColor"
                  stroke-width="0"
                  viewBox="0 0 512 512"
                  width="16"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M451 374c-15.88-16-54.34-39.35-73-48.76-24.3-12.24-26.3-13.24-45.4.95-12.74 9.47-21.21 17.93-36.12 14.75s-47.31-21.11-75.68-49.39-47.34-61.62-50.53-76.48 5.41-23.23 14.79-36c13.22-18 12.22-21 .92-45.3-8.81-18.9-32.84-57-48.9-72.8C119.9 44 119.9 47 108.83 51.6A160.15 160.15 0 0083 65.37C67 76 58.12 84.83 51.91 98.1s-9 44.38 23.07 102.64 54.57 88.05 101.14 134.49S258.5 406.64 310.85 436c64.76 36.27 89.6 29.2 102.91 23s22.18-15 32.83-31a159.09 159.09 0 0013.8-25.8C465 391.17 468 391.17 451 374z"
                    fill="none"
                    stroke-miterlimit="10"
                    stroke-width="32"
                  />
                </svg>
              </button>
            </div>
            <div
              class="footer__groupQuestions"
            >
              <a
                class="footer__linkAsk"
                href="https://ask.ivi.ru/?_gl=1*fa2mrd*_ga*MTMxNTY0MzcwMS4xNjgxMjE0Mzc5*_ga_GETQ4387MJ*MTY4MTU2Njc5MS44LjEuMTY4MTU2NzI1My4zLjAuMA.."
              >
                ask.ivi.ru
              </a>
              <span>
                Ответы на вопросы
              </span>
            </div>
          </div>
          <div
            class="footer__column"
          >
            <a
              class="footer__columnRight"
              href="https://www.ivi.ru/subscribe?redirect_url=%2Fwatch%2F504525"
            >
              <div
                class="footer__widget"
              >
                <svg
                  color="#fff"
                  fill="currentColor"
                  height="60"
                  stroke="currentColor"
                  stroke-width="0"
                  style="color: rgb(255, 255, 255);"
                  viewBox="0 0 24 24"
                  width="60"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M0 0h24v24H0z"
                    fill="none"
                  />
                  <path
                    d="M16.5 12A4.5 4.5 0 0014 7.97v2.21l2.45 2.45c.03-.2.05-.41.05-.63zm2.5 0c0 .94-.2 1.82-.54 2.64l1.51 1.51A8.796 8.796 0 0021 12c0-4.28-2.99-7.86-7-8.77v2.06c2.89.86 5 3.54 5 6.71zM4.27 3L3 4.27 7.73 9H3v6h4l5 5v-6.73l4.25 4.25c-.67.52-1.42.93-2.25 1.18v2.06a8.99 8.99 0 003.69-1.81L19.73 21 21 19.73l-9-9L4.27 3zM12 4L9.91 6.09 12 8.18V4z"
                  />
                </svg>
              </div>
              <span
                class="footer__widgetText"
              >
                Смотрите фильмы, сериалы и мультфильмы без рекламы
              </span>
            </a>
          </div>
        </div>
        <div
          class="footer__downBlock"
        >
          <div
            class="footer__downBlock_left"
          >
            <div
              class="footer__downBlock_group"
            >
              <a
                class="footer__smartBtn"
                href="https://apps.apple.com/RU/app/id455705533?mt=8"
              >
                <img
                  alt="apple"
                  data-nimg="1"
                  decoding="async"
                  height="20"
                  loading="lazy"
                  src="https://solea-parent.dfs.ivi.ru/picture/ffffff,ffffff/appleLogo.svg"
                  style="color: transparent;"
                  width="20"
                />
                <div>
                  <p
                    class="footer__smartBtn_title"
                  >
                    Загрузить в
                  </p>
                  <p
                    class="footer__smartBtn_text"
                  >
                    App Store
                  </p>
                </div>
              </a>
              <a
                class="footer__smartBtn"
                href="https://apps.apple.com/RU/app/id455705533?mt=8"
              >
                <img
                  alt="apple"
                  data-nimg="1"
                  decoding="async"
                  height="20"
                  loading="lazy"
                  src="https://solea-parent.dfs.ivi.ru/picture/ffffff,ffffff/googlePlayLogo.svg"
                  style="color: transparent;"
                  width="20"
                />
                <div>
                  <p
                    class="footer__smartBtn_title"
                  >
                    Доступно в
                  </p>
                  <p
                    class="footer__smartBtn_text"
                  >
                    Goggle Play
                  </p>
                </div>
              </a>
              <a
                class="footer__smartBtn"
                href="https://www.ivi.ru/pages/tvsmart/"
              >
                <svg
                  fill="currentColor"
                  height="20"
                  stroke="currentColor"
                  stroke-width="0"
                  viewBox="0 0 24 24"
                  width="20"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M0 0h24v24H0z"
                    fill="none"
                  />
                  <path
                    d="M20 3H4c-1.1 0-2 .9-2 2v11c0 1.1.9 2 2 2h6l-2 2v1h8v-1l-2-2h6c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2"
                  />
                </svg>
                <div>
                  <p
                    class="footer__smartBtn_title"
                  >
                    Смотрите на
                  </p>
                  <p
                    class="footer__smartBtn_text"
                  >
                    Smart TV
                  </p>
                </div>
              </a>
              <a
                class="footer__smartBtn"
                href="https://apps.apple.com/RU/app/id455705533?mt=8"
              >
                <img
                  alt="apple"
                  data-nimg="1"
                  decoding="async"
                  height="20"
                  loading="lazy"
                  src="https://solea-parent.dfs.ivi.ru/picture/ffffff,ffffff/appleLogo.svg"
                  style="color: transparent;"
                  width="20"
                />
                <div>
                  <p
                    class="footer__smartBtn_text"
                  >
                    Все устройства
                  </p>
                </div>
              </a>
            </div>
            <div>
              <p>
                © 2023 ООО «Иви.ру»
              </p>
              <p>
                HBO ® and related service marks are the property of Home Box Office, Inc
              </p>
            </div>
          </div>
          <div
            class="footer_downBlock_right"
          >
            <button
              class="footer__btn border"
            >
              <img
                alt="vk"
                data-nimg="1"
                decoding="async"
                height="16"
                loading="lazy"
                src="https://solea-parent.dfs.ivi.ru/picture/ffffff,ffffff/social_vkontakte.svg"
                style="color: transparent;"
                width="16"
              />
            </button>
            <button
              class="footer__btn border"
            >
              <img
                alt="od"
                data-nimg="1"
                decoding="async"
                height="16"
                loading="lazy"
                src="https://solea-parent.dfs.ivi.ru/picture/ffffff,ffffff/social_odnoklassniki.svg"
                style="color: transparent;"
                width="16"
              />
            </button>
            <button
              class="footer__btn border"
            >
              <img
                alt="twitter"
                data-nimg="1"
                decoding="async"
                height="16"
                loading="lazy"
                src="https://solea-parent.dfs.ivi.ru/picture/ffffff,ffffff/social_twitter.svg"
                style="color: transparent;"
                width="16"
              />
            </button>
            <button
              class="footer__btn border"
            >
              <img
                alt="viber"
                data-nimg="1"
                decoding="async"
                height="16"
                loading="lazy"
                src="https://solea-parent.dfs.ivi.ru/picture/ffffff,ffffff/social_viber.svg"
                style="color: transparent;"
                width="16"
              />
            </button>
            <button
              class="footer__btn border"
            >
              <img
                alt="linkendin"
                data-nimg="1"
                decoding="async"
                height="16"
                loading="lazy"
                src="https://solea-parent.dfs.ivi.ru/picture/ffffff,ffffff/social_linkedin.svg"
                style="color: transparent;"
                width="16"
              />
            </button>
            <button
              class="footer__btn border"
            >
              <img
                alt="telegram"
                data-nimg="1"
                decoding="async"
                height="16"
                loading="lazy"
                src="https://solea-parent.dfs.ivi.ru/picture/ffffff,ffffff/social_telegram.svg"
                style="color: transparent;"
                width="16"
              />
            </button>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
